packages/neuvector/generated-changes/patch/Chart.yaml.patch

```diff
@@ -0,0 +1,32 @@
+--- charts-original/Chart.yaml
++++ charts/Chart.yaml
+@@ -1,10 +1,28 @@
++annotations:
++  catalog.cattle.io/auto-install: neuvector-crd=match
++  catalog.cattle.io/certified: rancher
++  catalog.cattle.io/display-name: NeuVector
++  catalog.cattle.io/kube-version: '>=1.18.0-0'
++  catalog.cattle.io/namespace: cattle-neuvector-system
++  catalog.cattle.io/os: linux
++  catalog.cattle.io/permit-os: linux
```

Comment:
this should be `catalog.cattle.io/permits-os: linux,windows` and you will need to make sure the appropriate node-selectors/tolerations are set so the neuvector pods are not scheduled on Windoes nodes.

Reply:
we agreed with @cbron to use only linux for permit-os now because Neuvector supports only Linux nodes.

Reply:
Main question here is how NV reacts to a cluster with Windows nodes, since unlike most apps we have, it needs to interact with those nodes. Since we don't know for sure, I think its best to hold off for now on allowing that use case.

Reply:
@cbron We should still add windows to it in order to make neuvector deployable in a hybrid RKE2 cluster. Take for example the CIS benchmark chart which does not support Windows nodes, but you can still deploy it in a hybrid cluster due to these node-selectors/tolerations:

https://github.com/rancher/charts/blob/eaba8696d0e569a64c45ef7f610b501528daae9d/charts/rancher-cis-benchmark/2.0.3/templates/deployment.yaml#L44-L51

https://github.com/rancher/charts/blob/eaba8696d0e569a64c45ef7f610b501528daae9d/charts/rancher-cis-benchmark/2.0.3/templates/_helpers.tpl#L14-L27

Reply:
We could try it. My main point above was that those other apps just deal with the linux nodes they get scheduled on. Where as NeuVector might interact with the traffic of all nodes in the hybrid cluster, and thats the real unknown part, how NeuVector reacts in a hybrid cluster at all, regardless of which node in the cluster it gets scheduled on.

Reply:
We do have partial deployment cases, where not all nodes have enforcers running. The network rules are learned/defined and enforced accordingly. Having said that, the network function works the best in full deployment. I suggest we disable neuvector in mixed environment for now, and update the chart later.